tests/unit/modules/test_ansiblegate.py

```diff
@@ -75,65 +70,71 @@ class Module(object):
         """
 
         with patch.object(ansible, '_resolver', self.resolver), \
-            patch.object(ansible._resolver, 'load_module', MagicMock(return_value=Module())):
+                patch.object(ansible._resolver, 'load_module',
+                             MagicMock(return_value=Module())):
             ret = ansible.help('dummy')
-            assert sorted(ret.get('Available sections on module "{0}"'.format(
-                Module().__name__))) == ['one', 'two']
-            assert ret.get('Description') == 'describe the second part'
+            self.assertEqual(sorted(ret.get('Available sections on module "{0}"'
```

Comment:
@twangboy Is the use of pytest notation vs unittest notation breaking tests in windows? Because we should be able to keep the pytest style in this file.

Reply:
I've just seen the `assertEqual` and thought it was it was the preferred method... I don't know the difference.